### ISSUE TITLE
The FX mixer faders

### ISSUE THREAD

**Issue by Umcaruje** [HDDigitizerMusic/lmms-alt-theme#22]
Ok, so in your current theme, the faders of the FX mixer look like this:
![screenshot from 2016-03-28 23 32 37](https://cloud.githubusercontent.com/assets/6282045/14090982/bcdc2a04-f53d-11e5-80a3-ec8c369bc7e7.png)

With those 2 thin lines representing the DB readout. Now, I'm sure this looks nice from a design perspective, but for producers, those readouts are a bit useless, since they are small, and go under the fader, and you can't see an exact readout. Now, also (and this is a problem with the main theme too) you can't see when you are clipping. And this is a huge issue for producers. 

Now, what @stakeoutpunch did in the theme on which we worked on is actually measure the point where 0db is in the mixer, and then he later marked that pixel. that looks like this:
![screenshot from 2016-03-28 23 43 08](https://cloud.githubusercontent.com/assets/6282045/14091157/d979a528-f53e-11e5-92fc-50f31902c405.png)

And I think something like that should be incorporated in the main theme as well, but with a different touch. @michaelgregorius changed the 0db line over in https://github.com/LMMS/lmms/pull/2672, so we'll need reassessing, but generally the gradient should look something like this (behold my amazing mockup skills):
![hello its me](https://cloud.githubusercontent.com/assets/6282045/14091356/13c131c8-f540-11e5-9e1e-57ec2552fd0e.png)

So TL;DR: The dB readouts aren't very practical since they are small, and users should know exactly when they are clipping.

**Comment by StakeoutPunch** [comment]
![fader_background](https://cloud.githubusercontent.com/assets/6579031/14091478/201fea5c-f506-11e5-9477-4bef36aeaa2e.png)
![fader_leds](https://cloud.githubusercontent.com/assets/6579031/14091484/2adbe7f2-f506-11e5-99a1-d56a89272131.png)

Here are the two components of the mixer in the theme @Umcaruje is talking about.

**Comment by RebeccaDeField** [comment]
Thanks for the valuable feedback! I'm on it :)

**Comment by RebeccaDeField** [comment]
@Umcaruje Just read over the [LMMS#2672](https://github.com/LMMS/lmms/pull/2672) issue. It's easy enough that I'm going to update my design on this theme so that it is more usable for producers for the time being.

If we end up using CSS or perfecting the technical details I can just update/remove the graphics accordingly. :+1:

**Comment by Umcaruje** [comment]
:+1:

**Comment by RebeccaDeField** [comment]
My updated design:
![faderknob](https://cloud.githubusercontent.com/assets/7960169/14224010/ed6554dc-f841-11e5-9b85-4c7c0e0811e8.png)

Now that this design is more usable for producers, I think that we should close the issue here and continue any discussion about changing how the fader knobs work on the main issue tracker.

**Comment by StakeoutPunch** [comment]
@RebeccaDeField  Perfect!

**Comment by RebeccaDeField** [comment]
@StakeoutPunch Thank you! We couldn't have done it without your contribution :) :+1:
